db/agents.yml

```diff
@@ -39,7 +39,11 @@ agents:
     name: null
     type: Mobile App
     os: Android
-  - regex: ^Breaker.+CFNetwork
+  - regex: ^Bose/
+    name: Bose
+    type: Smart Home
+    os: Amazon OS #???
+  - regex: ^Breaker/iOS
```

Comment:
I no longer see any CFNetwork in Breaker user agents:

```
Breaker/iOS (https://breaker.audio)
Breaker/iOS
Breaker/0.7.10 (1034)
Breaker/iOS/0.8.0 (1080)
Breaker/0.7.7 (1003)
Breaker/0.7.5 (947)
```

The top 2 are the iOS ones ... the others are Android.